core/utils/poly-look/src/visualisations/charts/bar-charts/horizontalBarChart/horizontalBarChart.js

```diff
@@ -237,13 +237,16 @@ export class HorizontalBarChart extends Chart {
       .attr("fill", this._barValueColor);
   }
 
-  _updateExistingBarValues(barValues) {
-    barValues
+  _updateExistingBarValues(barGroups) {
+    barGroups
+      .selectAll("bar-value")
       .attr("y", (d) => this._setValueYAttribute(d))
       .attr("class", "bar-value")
       .attr("text-anchor", "end")
       .attr("x", (d) => this._xScale(d.value) - barValueMargin)
       .text((d) => d.value)
+      .attr("fill", "transparent")
+      .style("font-size", fontSizeInBar)
```

Comment:
We previously didn't set `fill` or `font-size`, but that's not the fix, is it?

Reply:
nope just keeping it similar to the entering function